### PR TITLE
Updated links from docs.iota.org to wiki.iota.org

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

Updated links from docs.iota.org to wiki.iota.org

## Type of change

- Bug fix (a non-breaking change which fixes an issue)
- Documentation Fix

## How the change has been tested

Changes were tested locally using node 12

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ If your feature has a public-facing API, please consider the following:
 <summary>Contribute to the documentation :black_nib:</summary>
 <br>
 
-The Streams documentation is hosted on https://docs.iota.org, which is built from content in the [documentation](https://github.com/iotaledger/documentation) repository.
+The Streams documentation is hosted on https://wiki.iota.org, which is built from content in the [documentation](https://github.com/iotaledger/documentation) repository.
```
Comment (arnauorriols) [inline .github/CONTRIBUTING.md:112]: https://github.com/iotaledger/documentation is deprecated, isn't it? can you update this phrase while at it?

**Review by arnauorriols — CHANGES_REQUESTED** [review]
`.github/workflows/deploy_docs_to_github_pages.yml` and `documentation/docusaurus.config.js` also contain references to docs.iota.org

Besides that and the other comment. I'm planning to merge once those links are operative, unless you tell me otherwise!

**Comment by arnauorriols** [comment]
Resolves #114

**Comment by lucas-tortora** [comment]
> `.github/workflows/deploy_docs_to_github_pages.yml` and `documentation/docusaurus.config.js` also contain references to docs.iota.org
> 
> Besides that and the other comment. I'm planning to merge once those links are operative, unless you tell me otherwise!

Updated.  These weren't changed because these aren't really used anymore as the wiki handles the config and publishing, but we're keeping them in place until we finally remove docs.iota entirely.

**Review by arnauorriols — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (arnauorriols) [reply .github/CONTRIBUTING.md]: Thanks, much better!

**Pull request merged**